functions/influxdb+grafana.bash

```diff
@@ -256,6 +263,11 @@ grafana_install(){
   fi
 
   echo -n "$(timestamp) [openHABian] Setting up Grafana service... "
+  cond_redirect mkdir -p /var/log/grafana /opt/zram/log.bind/grafana
+  cond_redirect chown grafana /var/log/grafana /opt/zram/log.bind/grafana
```

Comment:
yes will modularize
Trouble with your sed proposal is what if someone removes+reinstalls zram that's why I'd still prefer override.conf
Or you would need to extend zram install to check for all tools that use it
btw that's (I think)
mosquitto
InfluxDB+Grafana
Samba
exim

Reply:
> Trouble with your sed proposal is what if someone removes+reinstalls zram that's why I'd still prefer override.conf

That should not be an issue, we already do it with FIND3 and ztab. Same principle here. In the upstream repo samba & openhab are already taken care of. Exim can just be left alone as there is no extra configuration needed. IDK about MQTT though.

Reply:
Dunno if I get that right. Do you mean you (statically) add all services as Before= to your zram repo ?
That would turn that repo into something that no longer is independent of openHAB, wouldn't it.
Exim also logs to /var/log/something so potentially is affected as well.
And mosquitto is the main reason. It's "fixed" with an override.conf too but if you want a generic solution with those .confs (and I agree it would make sense to go for a generic solution) then we must find a solution for that in the first place works with mosquitto.